login-frontend/src/App.tsx

```diff
@@ -41,38 +40,33 @@ export function App() {
           credentials: 'include',
         })
       ).json();
-      const credentialResponse = await new Promise(async (resolve, reject) => {
+      const getCredentialFromExtension = await new Promise(async (resolve, reject) => {
         try {
-          await session.listen(async (credentialResponse) => {
-            resolve(credentialResponse);
+          await session.listen(async (credential) => {
+            resolve(credential);
           });
           await session.send(credentialRequirements);
         } catch (e) {
           reject(e);
         }
       });
 
-      let url = '/api/v1/credentials';
-      // get redirect uri query
-      const redirectUri = new URLSearchParams(window.location.search).get('redirect');
```

Comment:
Does this do anything? Based on what we say in the rust code?

Reply:
no it doesn't. Also `PostCredentialQueryParameter` is never used in the rust code